.github/workflows/run_examples_local.yml

```diff
@@ -8,8 +8,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [3.6]
-        os: [ubuntu-18.04]
+        python-version: [3.6, 3.7, 3.8]
+        os: [ubuntu-latest, macos-latest]
```

Comment:
How does this matrix make sense when we do none of the interesting stuff for macOS anyway? :)

Reply:
we run it against a local ert-storage solution (sqlite)

Reply:
the polynomial demo is ran on macos with sqlite